rescheduler.go

```diff
@@ -96,11 +96,11 @@ func main() {
 	// Add nodes labels as flags
 	flags.StringVar(&nodes.OnDemandNodeLabel,
 		"on-demand-node-label",
-		"node-role.kubernetes.io/worker",
+		"kubernetes.io/role=worker",
```

Comment:
I think it would be good to add some validation when the rescheduler starts to check the value of this flag once parsed, if it has no `=`, fine, if it has 1 `=` fine, if it has more than one, fail and print a useful message explaining the expected input for the flag? Ditto for the `spot-node-label` flag too, WDYT?

Reply:
Yep, sounds like sensible sanity checking